pkg/metrics/metrics.jsx

```diff
@@ -758,7 +758,7 @@ class CurrentMetrics extends React.Component {
                            value={top_cores[0][1]}
                            className="current-top-cpu-usage pf-m-sm"
                            min={0} max={100}
-                           variant={ top_cores[0][1] > 90 ? ProgressVariant.danger : ProgressVariant.info }
+                           variant={ top_cores[0][1] > 90 ? ProgressVariant.danger : top_cores[0][1] >= 80 ? ProgressVariant.warning : ProgressVariant.info }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19217-20230822-113626-06b76265-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)